src/System.Runtime.Serialization.Formatters/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Runtime.Serialization.Formatters:
+TypesMustExist : Type 'System.Runtime.Serialization.SerializationEventHandler' does not exist in the implementation but it does exist in the contract.
```

Comment:
@ViktorHofer should these be exposed in the ref?

Reply:
@morganbr changed that type's visibility to public to unblock serialization with uapaot. It's fine being not exposed in the ref assembly.

Reply:
@weshaggard any chance you coudl include little comments in the baseline files so we dont have to read this PR discussion months from now to figure out why stuff is in it?

Reply:
That is the plan. As I get confirmation from the owners I'm adding comments to the baseline or fixing issues (see the additional commits for examples of this).